packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -1,30 +1,24 @@
 <template>
 	<div :class="formGroupStyle">
-		<div
-			:class="decreaseButtonStyle"
-			@click="setCurrentValue(currentValue - step)"
-		>
+		<div :class="decreaseButtonStyle" @click="decrementValue">
```

Comment:
ik it's not on you
but changing the background color of a div when it has an icon shouldn't really do anything
does it?
so the kt-input_number--disabled would change the background color of the div but since it contains the icon, it won't do anything

soo 
move the class one step below and since the yoco icons are svgs, change the color property instead to light-gray

i'm looking to see if we have this implemented somewhere already

Reply:
kk